Cargo.toml

```diff
@@ -11,7 +11,6 @@ repository = "https://github.com/paritytech/extended-parachain-template/"
 [workspace]
 members = [
 	"node",
-	"pallets/*",
```

Comment:
Not sure if removing `pallets` would break anything as I don't remember if having dir `pallets` is necessary in any `new` commands. Would recommend a manual run to verify that this is indeed okay.

Reply:
It doesn't requiere it now. However a cool feature would be now when adding `pop new pallet` inside a node create a pallet folder, create the new pallet in this folder and add this line code in Cargo.toml.

Reply:
Added the feature: https://github.com/r0gue-io/pop-cli/issues/58